Assignment/training2/src/task1/Components/Modal/Modal.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import './Modal.css';
+
+function Modal({open}) {
+
+    return (<>{
+        open ? 
```

Comment:
@lehuy1012  the modal should not destroy children or DOM element when the modal is close

and you have not supported children prop how can I use your modal and custom the modal body, modal footer...

Reply:
missing the prop onHideModal

Reply:
fixed